src/mumble/Overlay_win.cpp

```diff
@@ -80,8 +94,10 @@ void OverlayPrivateWin::setActive(bool active) {
 
 		if (m_active) {
 			startHelper(m_helper_process);
+			startHelper(m_helper64_process);
```

Comment:
Don't we need some check on whether we actually are on a x64 capable platform? Or at least some facilities to catch/handle the resulting error.